app/forms/BusinessRegistrationForms.scala

```diff
@@ -37,6 +37,7 @@ object BusinessRegistrationForms {
 
   val NonUkPostCodeRegex = "^[a-zA-Z0-9]{1,10}+(?: [a-zA-Z0-9]{2,10})?$"
   val countryUK = "GB"
+  private val postcodeFormatPattern: String = "[+.:_,;=(){}\\[\\]\\-\\^\\*]"
```

Comment:
I think we should use "[^A-Za-z0-9 ]". This only allows upper or lowercase alphanumeric and space and removes everything else